lib/common/ipc_attrd.c

```diff
@@ -246,8 +246,9 @@ pcmk__attrd_api_clear_failures(pcmk_ipc_api_t *api, const char *node,
         op_desc = "operations";
     }
 
-    crm_debug("Asked pacemaker-attrd to clear failure of %s %s for %s on %s: %s (%d)",
-              interval_desc, op_desc, (resource? resource : "all resources"),
+    crm_debug("Asked %s to clear failure of %s %s for %s on %s: %s (%d)",
+              pcmk_ipc_name(api, true), interval_desc, op_desc,
+              (resource? resource : "all resources"),
               (node? node : "all nodes"), pcmk_rc_str(rc), rc);
```

Comment:
You could use pcmk__s for both resource and node here.

Reply:
might as well